Description: Remove GTK icon cache rebuilding.
 This removes the rebuilding of the GTK icon cache after installing the core.
 It is not needed in any distro with hicolor-icon-theme >= 0.10-2
Forwarded: not-needed
Author: Jason Heeris <jason.heeris@gmail.com>

diff -Nur -x '*.orig' -x '*~' rabbitvcs-0.13/setup.py rabbitvcs-0.13.new/setup.py
--- rabbitvcs-0.13/setup.py	2010-01-30 22:39:56.000000000 +0800
+++ rabbitvcs-0.13.new/setup.py	2010-01-30 22:50:29.000000000 +0800
@@ -141,10 +141,11 @@
 #
 
 # Make sure the icon cache is deleted and recreated
-if sys.argv[1] == "install":
-    print "Running gtk-update-icon-cache"
-    
-    subprocess.Popen(
-        ["gtk-update-icon-cache", icon_theme_directory], 
-        stdout=subprocess.PIPE
-    ).communicate()[0]
+# Debian: this is not needed since hicolor-icon-cache (0.10-2)
+# if sys.argv[1] == "install":
+#     print "Running gtk-update-icon-cache"
+#     
+#     subprocess.Popen(
+#         ["gtk-update-icon-cache", icon_theme_directory], 
+#         stdout=subprocess.PIPE
+#     ).communicate()[0]
